jekyll/_cci2/gh-bb-integration.md

```diff
@@ -296,20 +296,24 @@ by following this workaround:
 1. In the CircleCI application,
 go to your project's settings.
 
-2. Navigate to the **Checkout SSH Keys** page.
+2. Navigate to the **SSH Keys** page.
 
-3. Right-click the **Create `<username>` user key** button
-and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/bb_user_key.png)
+3. Right-click the **Add User Key** button and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/new_bb_user_key.png)
+    - **For users of the legacy CircleCI UI:** Right-click the **Create `<username>` user key** button and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/bb_user_key.png)
```

Comment:
Sorry, should have said the legacy UI is only used by CircleCI server customers now, and Bitbucket isn't supported in that product line, so these references can be removed too :-)